Solved issue #10

tlegen-k opened the pull request:
Added description of the helm installation using helm chart repository hosted on github to README file

tlegen-k committed:
Update README.md

Update README.md file according to issue #10.

tlegen-k committed:
Merge pull request #1 from tlegen-k/tlegen-k-patch-1

Update README.md

mstein11 commented on README.md:
```diff
@@ -17,6 +17,11 @@ You can install the chart by downloading this repository and running the helm in
 
 By running the above command you will install the docker-hub-rate-limit-exporter into your cluster. It will expose the dockerhub limits in the prometheus format.
 
+The helm chart can be installed installed through a helm chart repository hosten on a github page in this repository. To install follow the next steps:
```
I think you have put the word "installed" twice. Can you remove one? :-)

mstein11 replied:
Also, I think it would make sense to remove line 13 to 16, since we don't want people to install this chart through git clone anymore. So maybe replace line 13 to 16 with your additions?

tlegen-k replied:
I updated the README file according to your request. But i can't make new pull request. Can you see the changes?

mstein11 replied:
Yes I can see your changes, thanks a lot :-)
Once you have opened a pull requests all changes on the branch which you request to be merge will be automatically added to the pull request, there is no need for opening another pull requests.
 
The updated readme looks good now, however I think we also have to update the readme in the `chart/` folder (https://github.com/viadee/docker-hub-rate-limit-exporter/blob/main/chart/README.md). I didn't think about this while reviewing your changes initially, sorry. Once the Readme.md in the chart folder is updated I will press the merge button. :-)

tlegen-k replied:
Thanks for clarifying and suggestion. Edited accordingly. Apart from that if there are any products of viadee that i can contribute using c++, python or bash please let me know. Would love to be of any assistance.

mstein11 replied:
Thanks, the changes look good now! :-)

And thank you for offering further help. I'll talk to @fkoehne to see if we have some issues on one of our other open source projects. I am sure we will find something :-)

tlegen-k replied:
I'm glad to be helpful! Thank you!

fkoehne replied:
We are currently working on the design of a recommender system for XAI methods, which will be in python (the prototype already is). Is that interesting for you?

tlegen-k replied:
Hello, thanks for getting back. . The topic is quite new for me, but I would love to be of any help! Can you provide a link for repository?

fkoehne replied:
We hope to publish it soon.

mstein11 reviewed (changes requested):
Thank you for contributing! Do you mind fixing the small issues as described in the comments? :-)

tlegen-k committed:
Update README.md

tlegen-k committed:
Update README.md

Updated docker-hub-rate-limit-exporter/chart/README.md with new installation guide using  a helm chart repository hosted on a github page in this repository.

tlegen-k committed:
Update README.md

Corrected spelling mistake.
